mysite/templates/checkers/player_stats.html

```diff
@@ -31,6 +31,24 @@
   </div>
 </nav>
 
-<div> this page will show player states   </div>
+
+{% for winner in winners %}
+
+
+<div class="container">
+<div class="card">
+  <div class="card-header">
+    <b>Game number:<span style="color:green"><i>{{winner.game.game_id}}</i></span></b> <br>Created by: <b><i>{{winner.game.player1_username}}</i></b>
+  </div>
+  <div class="card-body">
+    <h5 class="card-title">Played against <b><i>{{winner.game.player2_username}}</i></b></h5>
+    <p class="card-text"><b><span style="color:red"><i>Winner:{{winner.winner_user}}</i></span></b></p>
```

Comment:
Can you make this part better, because it is currently showing winner in red. (just a suggestion)